python/pyxet/pyxet/cli.py

```diff
@@ -37,11 +38,7 @@ def _ltrim_match(s, match):
     ```
     Used to compute relative paths.
     """
-    if len(s) < len(match):
-        raise (ValueError(f"Path {s} not in directory {match}"))
-    if s[:len(match)] != match:
-        raise (ValueError(f"Path {s} not in directory {match}"))
-    return s[len(match):]
+    return os.path.relpath(s, match)
```

Comment:
Nit: This function probably should not be called ltrim_match anymore.

Reply:
Renamed to `_rel_path`